experiments/joy_stick/joy_stick.ino

```diff
@@ -1,9 +1,9 @@
 #include <Servo.h>
 #include <math.h>
 
-const int switchPin = A1;  // Joystick button
-const int pinX = A3; // Joystick X axis
-const int pinY = A2; // Joystick Y axis
+#define int switchPin A1;  // Joystick button
+#define int pinX A3; // Joystick X axis
+#define int pinY A2; // Joystick Y axis
```

Comment:
```suggestion
#define pinY A2; // Joystick Y axis
```
no data type here. You're not creating a new variable, you're just redirecting to the original value

Reply:
(same for the other #defines)

Reply:
Uhh, so it really happens on the lexical level?

Reply:
yes. If you run the preprocessor only over the file it'll literally replace every occurence of pinY with A2.
On that note, A2 is also just a constant defined, i.e., there's probably a statement `#define A2 2` somewhere in the arduino library. That means that even A2 doesn't have a data type, it just pastes a literal `2` everywhere